docs/how-to/code-generator-version-control.md

```diff
@@ -11,6 +11,10 @@ Amplication's code generator periodically releases new versions with bug fixes,
 
 While it's recommended to use the latest version, some teams may prefer having more control over when to upgrade the generated code.
 
+:::note
+Try Code Generator Version Control during your 14-day Enterprise plan trial for new workspaces.
```

Comment:
I wonder if we need to write here that it's available in Enterprise plan only, and then say that you can try it in your 14-day... 
WDYT?